$MirOS: ports/net/rtorrent/patches/patch-doc_rtorrent_rc,v 1.3 2007/05/08 23:30:48 tg Exp $
--- doc/rtorrent.rc.orig	Fri Feb  2 11:09:30 2007
+++ doc/rtorrent.rc	Wed May  9 01:19:19 2007
@@ -1,5 +1,7 @@
+# $MirOS$
+
 # This is an example resource file for rTorrent. Copy to
-# ~/.rtorrent.rc and enable/modify the options as needed. Remember to
+# ~/.etc/rtorrent.rc and enable/modify the options as needed. Remember to
 # uncomment the options you wish to enable.
 
 # Maximum and minimum number of peers to connect to per torrent.
@@ -19,11 +21,13 @@
 
 # Default directory to save the downloaded torrents.
 #directory = ./
+#directory = ~/BT/
 
 # Default session directory. Make sure you don't run multiple instance
 # of rtorrent using the same session directory. Perhaps using a
 # relative path?
 #session = ./session
+#session = ~/.etc/rtorrent.session
 
 # Watch a directory for new torrents, and stop those that have been
 # deleted.
@@ -73,6 +77,9 @@
 # plaintext to RC4 encryption after the encrypted handshake
 #
 # encryption = allow_incoming,enable_retry,prefer_plaintext
+#
+# MirPorts Framework: Hey, why not encrypt by default? It's nice!
+encryption = allow_incoming,try_outgoing,enable_retry
 
 #
 # Do not modify the following parameters unless you know what you're doing.
@@ -91,6 +98,11 @@
 # before forcing. Overworked systems might need lower values to get a
 # decent hash checking rate.
 #hash_max_tries = 10
+
+# http://libtorrent.rakshasa.no/ticket/14 - Hashing slow on OpenBSD
+hash_read_ahead = 4
+hash_interval = 10
+hash_max_tries = 2
 
 # Max number of files to keep open simultaniously.
 #max_open_files = 128
